app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :confirmable,
+         :recoverable, :rememberable, :trackable, :validatable
+  validates :name, :surname, presence: true,
+                             length: { in: 3..30 },
+                             format: { with: /\A[a-zA-Z]+\z/,
+                                       message: 'only allows latin letters' }
```

Comment:
Here we should use I18n and have a translation in the `locals/` folder. Probably will update it in another task.

Reply:
no one ever will change something in another task :)